### DISCUSSION TITLE
How to debug RSS Atom Feeds in docusaurus?

### DISCUSSION THREAD

**Discussion by tlyeung** [facebook/docusaurus#5915] [Q&A] [answered]
Hi,

I don't know if I have done something stupid.
Now my the feeds in my docusaurus page are gone.
There are no rss.xml/atom.xml pages in my site now.

Now my feed url is like this https://tutorial.docusaurus.io/blog/rss.xml
Page Not Found... blah blah blah 

It was like this before https://docusaurus.io/blog/rss.xml

In the docusaurus.config.js:, I have enabled the feed options.

``` javascript
blog: {
            feedOptions: {
                type: 'all',
                copyright: `Copyright © ${new Date().getFullYear()} ...`,
            },
            showReadingTime: false,
            blogSidebarTitle: 'All posts',
            blogSidebarCount: 'ALL',
}
```

And in the index.html pages, there are lines:
``` xml
<link rel="alternate" type="application/rss+xml" href="/blog/rss.xml" title="My RSS Feed">
<link rel="alternate" type="application/atom+xml" href="/blog/atom.xml" title="My Atom Feed">
```
However the rss.xml/atom.xml pages are not built.

Where else should I check?

Thanks.

**Comment by slorber** [comment, marked as answer]
This is a bug in beta.8 afaik, please upgrade

**Reply by tlyeung** [reply]
good to know it.
otherwise it may drive me crazy.
